.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: release
+
+on: release
+
+jobs:
+  release:
+    name: build and publish release assets
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: build release
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --all-features
+
+      - name: package release assets
+        run: |
+          mkdir _dist
+          cp README.md LICENSE target/release/krustlet-wasi target/release/krustlet-wascc _dist/
+          cd _dist
+          tar czf krustlet-${{ steps.get_version.outputs.VERSION }}-linux-amd64.tar.gz README.md LICENSE krustlet-wasi krustlet-wascc
```

Comment:
Any reason we can't just add in the macOS build right now?

Reply:
No particular reason. Ill see what I can do.

Reply:
done